src/science/canopy/cbl_dryLeaf.F90

```diff
@@ -15,33 +15,38 @@ SUBROUTINE dryLeaf( dels, rad, rough, air, met,                                &
 
     USE cable_def_types_mod
     USE cable_common_module
-USE cbl_photosynthesis_module,  ONLY : photosynthesis
-USE cbl_fwsoil_module,        ONLY : fwsoil_calc_std, fwsoil_calc_non_linear,           &
-                                     fwsoil_calc_Lai_Ktaul, fwsoil_calc_sli
-!data
-USE cable_surface_types_mod, ONLY: evergreen_broadleaf, deciduous_broadleaf    
-USE cable_surface_types_mod, ONLY: evergreen_needleleaf, deciduous_needleleaf
-USE cable_surface_types_mod, ONLY: c3_grassland, tundra, c3_cropland  
-   
-! maths & other constants
-USE cable_other_constants_mod, ONLY : CLAI_THRESH  => LAI_THRESH
-! physical constants
-USE cable_phys_constants_mod, ONLY : CTFRZ   => TFRZ
-USE cable_phys_constants_mod, ONLY : CDHEAT  => DHEAT
-USE cable_phys_constants_mod, ONLY : CRGAS   => RGAS
-USE cable_phys_constants_mod, ONLY : CCAPP   => CAPP
-USE cable_phys_constants_mod, ONLY : CRMAIR  => RMAIR
-USE cable_phys_constants_mod, ONLY : density_liq
-! photosynthetic constants
-USE cable_photo_constants_mod, ONLY : CMAXITER  => MAXITER ! only integer here
-USE cable_photo_constants_mod, ONLY : CTREFK => TREFK
-USE cable_photo_constants_mod, ONLY : CGAM0  => GAM0
-USE cable_photo_constants_mod, ONLY : CGAM1  => GAM1
-USE cable_photo_constants_mod, ONLY : CGAM2  => GAM2
-USE cable_photo_constants_mod, ONLY : CRGSWC => RGSWC
-USE cable_photo_constants_mod, ONLY : CRGBWC => RGBWC
-
-IMPLICIT NONE
+    USE cbl_photosynthesis_module,  ONLY : photosynthesis
+    USE cbl_fwsoil_module,          ONLY : fwsoil_calc_std,                   &
```

Comment:
since we're now being more insistent on transpiration coming from liquid water we need to revisit these `fwsoil` calculations - some are using `%wb`, some `%wbliq`.

Reply:
Done.

Reply:
good